emu/processes/wps_dry_run.py

```diff
@@ -0,0 +1,58 @@
+from pywps import Process, LiteralInput, LiteralOutput
+from pywps.inout.literaltypes import AllowedValue
+from pywps.app.Common import Metadata
+
+import logging
+LOGGER = logging.getLogger("PYWPS")
+
+
+class SimpleDryRun(Process):
+    """An example of implementing a dry-run to protect your process from unwanted resource usage."""
+    def __init__(self):
+        inputs = [
+            LiteralInput('dry_run', 'Dry run mode. Default false',
+                         data_type='boolean',
+                         default=False,),
+            LiteralInput('num_files', 'Number of Files (Limit 10)',
+                         abstract='How many files do you want to download? The limit is 10',
+                         data_type='integer',
```

Comment:
Why not use an allowed value with maxval ?

Reply:
done.